website/cue/reference/cli.cue

```diff
@@ -280,7 +281,7 @@ cli: {
 				}
 				"limit": {
 					_short:      "l"
-					description: "Sample log events to the provided limit"
+					description: "Number of log events to sample each interval"
```

Comment:
Nit: While I think this is clearer than what we had, this message suggests to me that I'll always get `limit` back. In reality,`limit` is a maximum value; it could be anything up to it.

I wonder if adding _(max)_ after the sentence better implies that?